src/core/stdc/config.d

```diff
@@ -15,17 +15,35 @@ module core.stdc.config;
 
 version (StdDdoc)
 {
+    private
+    {
+        version (Posix)
+            enum isPosix = true;
+        else
+            enum isPosix = false;
+        static if (isPosix && (void*).sizeof > int.sizeof)
+        {
+            alias ddoc_long = long;
```

Comment:
I don't see why we need these itermediate `ddoc_` aliases here.  Can't we just declare `c_long` and `c_ulong` here.